src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -291,6 +339,41 @@ def reweight(self, svc_id, weight):
             id=int(svc_id),
             weight=float(weight))
 
+    @Endpoint('GET')
+    @ReadPermission
+    @raise_if_no_orchestrator([OrchFeature.DAEMON_LIST])
+    @handle_orchestrator_error('host')
+    def deployment_options(self):
```

Comment:
don't we need to mark this endpoint experimental?

Reply:
can do, will do.